test/bulk2.test.ts

```diff
@@ -282,6 +282,8 @@ it('should call bulk api from invalid session conn with refresh fn, and return r
     },
   });
 
+  conn2.bulk2.pollTimeout = 90000;
+
```

Comment:
when calling `loadAndWait` in L291 it was using the default 10s timeout for polling:
https://github.com/jsforce/jsforce/actions/runs/7628707431/job/20780447068#step:8:233

set to 90s like in other tests.